release/release_gold.sh

```diff
@@ -40,25 +65,30 @@ function main {
 
 	lc_time_run promote_packages
 
+	lc_time_run tag_release
+
 	promote_boms
 
 	update_product_info_json
+
 }
 
 function print_help {
-	echo "Usage: LIFERAY_RELEASE_RC_BUILD_TIMESTAMP=<timestamp> LIFERAY_RELEASE_VERSION=<version> ${0}"
-	echo ""
-	echo "The script reads the following environment variables:"
-	echo ""
+    echo "Usage: (LIFERAY_RELEASE_FORK_NAME=<username/repository_name>) LIFERAY_RELEASE_GITHUB_PAT=<Personal Access Token> LIFERAY_RELEASE_RC_BUILD_TIMESTAMP=<timestamp> LIFERAY_RELEASE_VERSION=<version> ${0}"
+    echo ""
+    echo "The script reads the following environment variables:"
+    echo ""
+	echo "    LIFERAY_RELEASE_FORK_NAME (optional): The repository owner's git username, defaults to 'liferay' and the repository's name"
+    echo "    LIFERAY_RELEASE_GITHUB_PAT: The user's Personal Access Token"
 	echo "    LIFERAY_RELEASE_NEXUS_REPOSITORY_PASSWORD: Nexus user's password"
 	echo "    LIFERAY_RELEASE_NEXUS_REPOSITORY_USER: Nexus user with the right to upload BOM files"
 	echo "    LIFERAY_RELEASE_PRODUCT_NAME (optional): Set to \"portal\" for CE. The default is \"DXP\"."
-	echo "    LIFERAY_RELEASE_RC_BUILD_TIMESTAMP: Timestamp of the build to publish"
-	echo "    LIFERAY_RELEASE_VERSION: DXP version of the release to publish"
+    echo "    LIFERAY_RELEASE_RC_BUILD_TIMESTAMP: Timestamp of the build to publish"
+    echo "    LIFERAY_RELEASE_VERSION: DXP version of the release to publish"
 	echo ""
-	echo "Example: LIFERAY_RELEASE_RC_BUILD_TIMESTAMP=1695892964 LIFERAY_RELEASE_VERSION=2023.q3.0 ${0}"
+    echo "Example: LIFERAY_RELEASE_FORK_NAME=<username/repository_name> LIFERAY_RELEASE_GITHUB_PAT=<personal access token> LIFERAY_RELEASE_RC_BUILD_TIMESTAMP=1695892964 LIFERAY_RELEASE_VERSION=2023.q3.0 ${0}"
 
-	exit "${LIFERAY_COMMON_EXIT_CODE_HELP}"
+    exit "${LIFERAY_COMMON_EXIT_CODE_HELP}"
```

Comment:
What was changed here?

Reply:
it got lower in lines because of the changes

Reply:
I don't understand, let's discuss it by slack.